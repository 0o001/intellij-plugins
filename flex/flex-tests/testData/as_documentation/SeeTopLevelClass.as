package com.foo {
/**
 * @see global#F<caret>oo
 */
public class Test {
}
}
