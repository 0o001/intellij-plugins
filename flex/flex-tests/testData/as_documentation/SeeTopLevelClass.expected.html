<PRE><b>Foo</b></PRE> Top level class docs